File: web3rpc/rpc-specs/code-samples/javascript/debug/vMTracing/traceBlockFromFile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const fileName = "/home/sotatek/block.rlp";

    provider.debug.traceBlockFromFile(fileName, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/debug/vMTracing/traceBlockFromFile.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

fileName = "/home/sotatek/block.rlp";

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.trace_block_from_file(fileName)

print(debug_response)
